extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -68,18 +71,33 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
     return this.views.get(variantAnalysisId);
   }
 
+  public async loadResults(variantAnalysisId: number, repositoryFullName: string): Promise<void> {
+    const variantAnalysis = this.variantAnalyses.get(variantAnalysisId);
+    if (!variantAnalysis) {
+      throw new Error(`No variant analysis with id: ${variantAnalysisId}`);
```

Comment:
Out of interest, how would this error surface to the user? Would it appear as a notification or be in a debug window or be completely hidden?

Reply:
Since this is called by [a command runner](https://github.com/github/vscode-codeql/blob/dc6ae6cc39e5f44f0ba60d68a51f54470529c1f9/extensions/ql-vscode/src/commandRunner.ts#L105-L150), this will show a warning notification and be logged (using the [`showAndLogWarningMessage`](https://github.com/github/vscode-codeql/blob/dc6ae6cc39e5f44f0ba60d68a51f54470529c1f9/extensions/ql-vscode/src/helpers.ts#L62-L66) function).